docs/concepts/etcd-cluster-resource-protection.md

```diff
@@ -0,0 +1,24 @@
+# Etcd Cluster Resource Protection
+
+`etcd-druid` provisions [kubernetes resources (a.k.a components)](etcd-cluster-components.md) for each `Etcd` cluster. Lifecycle of all of these components is managed by etcd-druid. To ensure that each component's specification is in line with the configured attributes defined in `Etcd` custom resource and to protect unintended changes done to any of these *managed components* a [Validating Webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/) is employed.
+
+[Etcd Components Webhook](https://github.com/gardener/etcd-druid/tree/55efca1c8f6c852b0a4e97f08488ffec2eed0e68/internal/webhook/etcdcomponents) is the *validating webhook* which prevents unintended *UPDATE* and *DELETE* operations on all managed resources. Following sections describe what is prohibited and in which specific conditions the changes are permitted.
+
+## Configure Etcd Components Webhook
+
+Prerequisite to enable the validation webhook is to [configure the Webhook Server](../deployment/configure-etcd-druid.md#webhook-server). Additionally you need to enable the `Etcd Components` validating webhook and optionally configure other options. You can look at all the options [here](../deployment/configure-etcd-druid.md#etcd-components-webhook).
+
+## What is allowed?
+
+Modifications to managed resources under the following circumstances will be allowed:
+
+* `Create` and `Connect` operations are allowed and no validation is done.
```

Comment:
what do you mean by `Connect` operations ?
are you referring to `get` the resource ?

Reply:
Please refer to https://github.com/gardener/etcd-druid/blob/master/internal/webhook/etcdcomponents/handler.go#L28
I could only find a single reference in the k8s repo:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go#L178